api/base/parsers.py

```diff
@@ -15,6 +17,29 @@ class JSONAPIParser(JSONParser):
     media_type = 'application/vnd.api+json'
     renderer_class = JSONAPIRenderer
 
+    def flatten_relationships(self, relationships):
+        """
+        Flattens relationships dictionary which has information needed to create related resource objects.
+
+        Validates that formatting of relationships dictionary is correct.
+        """
+        if not isinstance(relationships, dict):
+            raise ParseError()
+        related_resource = relationships.keys()[0]
```

Comment:
After discussing, we are not going to be creating multiple types of relationships simultaneously --  it's a design decision.  There is no use case for this and we are not going to log this as an issue.  Our method of creating relationships is a POST to /v2/nodes/{node_id}/contributors/ which does more than just make a connection between a node and a user, it creates a third item, called a contributor, which has additional attributes beyond the user. 

Creating multiple types of relationships would involve compound documents.  Creating a node while simultaneously creating a contributor and creating a node link is outside of our pattern.  

I will leave a note in the source code about how it only creates one type of relationship.